Assets/Scripts/Constellation.cs

```diff
@@ -55,18 +54,13 @@ public void ShowConstellationLines(bool show)
                     int endIndex = stars.FindIndex(star => star.GetComponent<StarComponent>().starData.Hipparcos == conn.endStarHipId);
                     if (startIndex >= 0 && endIndex >= 0)
                     {
-                        GameObject connectionLine = new GameObject();
-                        connectionLine.name = "ConnectionLine";
-                        connectionLine.transform.parent = this.transform;
-                        LineRenderer lineRenderer = connectionLine.AddComponent<LineRenderer>();
-                        lineRenderer.startWidth = lineWidth;
-                        lineRenderer.endWidth = lineWidth;
-                        lineRenderer.useWorldSpace = false;
-                        lineRenderer.SetPosition(0, new Vector3(stars[startIndex].transform.position.x, stars[startIndex].transform.position.y, stars[startIndex].transform.position.z));
-                        lineRenderer.SetPosition(1, new Vector3(stars[endIndex].transform.position.x, stars[endIndex].transform.position.y, stars[endIndex].transform.position.z));
-                        lineRenderer.material = lineMaterial;
-                        lineRenderer.material.color = lineColor;
-                        constellationLines.Add(connectionLine);
+                        if (constellationLinePrefab)
+                        {
+                            LineRenderer connectionLine = Instantiate(constellationLinePrefab, this.transform);
+                            connectionLine.SetPosition(0, new Vector3(stars[startIndex].transform.position.x, stars[startIndex].transform.position.y, stars[startIndex].transform.position.z));
+                            connectionLine.SetPosition(1, new Vector3(stars[endIndex].transform.position.x, stars[endIndex].transform.position.y, stars[endIndex].transform.position.z));
+                            constellationLines.Add(connectionLine.gameObject);
+                        }
```

Comment:
Did this cut back on the materials that we're being loaded?

Reply:
Yes, we've dropped back down in draw calls accordingly! No more instanced materials, just one material reused throughout the lines.